interpreter/statement.go

```diff
@@ -115,6 +124,9 @@ func (i *Interpreter) ProcessBlockStatement(statements []ast.Statement, ds Debug
 			state := i.ProcessReturnStatement(t)
 			return state, DebugPass, nil
 
+		case *ast.FallthroughStatement:
+			return FALLTHROUGH, DebugPass, nil
+
```

Comment:
I'm wondering why we handle fallthrough statement here  - it should be done in `ProcessSwitchStatement` because the `fallthrough` statement is just an internal state in `swtich` statement, not an VCL statement.

Reply:
It just seemed to me to be a straightforward way to indicate that a case should fall through since it doesn't require storing that information into the case AST struct or needing to check what the final statement was in the cases statement list. But I'll rework this to not have any internal switch logic in `ProcessBlockStatement`.

Reply:
Removed the handling of `fallthrough` from `ProcessBlockStatement`.